src/controllers/deliveryLog.js

```diff
@@ -1,24 +1,59 @@
+import { getCohortById } from '../domain/cohort.js'
+import { createDeliveryLogDb } from '../domain/deliveryLog.js'
 import { sendDataResponse } from '../utils/responses.js'
+import ERR from '../utils/errors.js'
 
-export const create = async (req, res) => {
-  const { date, cohort_id: cohortId, lines } = req.body
+export const createDeliveryLog = async (req, res) => {
+  const { cohort_id: cohortId, lines } = req.body
+  const { firstName, lastName } = req.user
 
-  return sendDataResponse(res, 201, {
-    log: {
-      id: 1,
-      cohort_id: cohortId,
-      date,
-      author: {
-        id: req.user.id,
-        first_name: req.user.firstName,
-        last_name: req.user.lastName
-      },
-      lines: lines.map((line, index) => {
-        return {
-          id: index + 1,
-          content: line.content
+  try {
+    if (!cohortId || !lines) {
+      throw Error(ERR.INCOMPLETE_REQUEST)
```

Comment:
Why are we throwing an error only to handle it in the catch? Couldn't we just check if the `cohortId` and `lines` exists prior to the try block and just return the response?